seed/mappings/mapper.py

```diff
@@ -1,7 +1,6 @@
 """
 :copyright: (c) 2014 Building Energy Inc
```

Comment:
I notice the old copyright notice is in the mapper.py file.
Looks like that file was missed when @pipermerriam made the pass to put in the new notice, or there's a versioning issue.